projects/lwan/request_fuzzer.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
+dict = http.dict
```

Comment:
don't need to declare dict here. just
create request_fuzzer.dict file and it will be automatically recognized as dict for request_fuzzer.

Reply:
Done.